src/System.Net.Requests/src/FxCopBaseline.cs

```diff
@@ -0,0 +1,4 @@
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Microsoft.Design", "CA2237", Scope = "type", Target = "System.Net.FtpWebRequest")]
+[assembly: SuppressMessage("Microsoft.Design", "CA2237", Scope = "type", Target = "System.Net.FtpWebResponse")]
```

Comment:
The ones i suppressed inline are ones that we probably won't fix. HttpRequestException doesn't have a serialization ctor in full framework.